bin/ask.js

```diff
@@ -1,29 +1,30 @@
 #!/usr/bin/env node
-/* eslint-disable global-require */
```

Comment:
please fix diffs so it is easier to see what changed.

Reply:
solved in new rev - crlf is the issue